src/Sentry.Unity/SentryOptionsUtility.cs

```diff
@@ -9,35 +10,66 @@ public static void SetDefaults(SentryUnityOptions options, IApplication? applica
         {
             application ??= ApplicationAdapter.Instance;
 
-            // 'Optimal' and 'Fastest' don't work on IL2CPP. Forcing 'NoCompression'.
+            options.Enabled = true;
+            options.Dsn = string.Empty;
+            options.CaptureInEditor = true;
             options.RequestBodyCompressionLevel = CompressionLevelWithAuto.NoCompression;
+            options.AttachStacktrace = false;
+            options.StackTraceMode = default;
```

Comment:
Can we please be explicit here? Even a comment on top of it (i.e: Enhanced isn't supported on IL2CPP)
```suggestion
            options.StackTraceMode = Original;
```